util/sys/storage/sys_darwin.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build darwin
```

Comment:
Please add an empty line with the license comment.

Reply:
```suggestion
// +build darwin linux
```
so that sys_linux.go can be merged into this file.